priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.TestRunner do
+  @moduledoc """
+  This module runs tests using `mix run`. For example:
+
+  mix run -e "LoadTest.TestRunner.run()" -- "deposits" "1" "5"
+
+  It accepts three arguments:
+  - test name
+  - transactions per seconds
+  - period in seconds
+
+  It fetches all configuration params from env vars.
+  """
+  alias LoadTest.TestRunner.Config
+
+  def run() do
+    {runner_module, config} = Config.parse()
+    result = Chaperon.run_load_test(runner_module, print_results: true, config: config)
+
+    case result.metrics["success_rate"][:mean] do
```

Comment:
slightly concern how scaleable is this approach. I do think we might assert on different conditions for different perf tests. For instance, in watcher info perf tests, I might be interested to fail when `p99 of API_X latency > Y` or when an `API error rate > Y%`. 

Probably need sth like a `verify_result_module` that we can parse out from Config too? (just like the runner_module)

Reply:
I added percentile as an optional fourth parameter https://github.com/omgnetwork/elixir-omg/pull/1745/commits/3071f826c1dc9c2d5c5e55b87efe823032373f85

Reply:
haha my concern is slightly different. I am just thinking not all test want to assert on same stuff. Probably some does not even care about error_rate. But I think we can iterate on this as you're going to continue working with perf anyway. I guess you'll figure the solution out when adding more tests with different conditions to assert.